components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -192,9 +324,67 @@ func (backend *ESClient) setDailyLatestToFalse(ctx context.Context, nodeId strin
 }
 
 func (backend *ESClient) DeleteProjectTag(ctx context.Context,
```

Comment:
Signature needs to change per #2389 -- see https://github.com/chef/automate/pull/2428/files#r354108267